src/main/java/uk/gov/hmcts/darts/arm/rpo/impl/ArmRpoApiImpl.java

```diff
@@ -236,7 +231,30 @@ public ArmAsyncSearchResponse addAsyncSearch(String bearerToken, Integer executi
 
     @Override
     public void saveBackgroundSearch(String bearerToken, Integer executionId, String searchName, UserAccountEntity userAccount) {
-        throw new NotImplementedException("Method not implemented");
+        var armRpoExecutionDetailEntity = armRpoService.getArmRpoExecutionDetailEntity(executionId);
+        armRpoService.updateArmRpoStateAndStatus(armRpoExecutionDetailEntity, ArmRpoHelper.saveBackgroundSearchRpoState(),
+                                                 ArmRpoHelper.inProgressRpoStatus(), userAccount);
+
+        StringBuilder errorMessage = new StringBuilder("Failure during ARM save background search: ");
+        SaveBackgroundSearchResponse saveBackgroundSearchResponse;
+        try {
+            SaveBackgroundSearchRequest saveBackgroundSearchRequest =
+                createSaveBackgroundSearchRequest(searchName, armRpoExecutionDetailEntity.getSearchId());
+            saveBackgroundSearchResponse = armRpoClient.saveBackgroundSearch(bearerToken, saveBackgroundSearchRequest);
+        } catch (FeignException e) {
+            // this ensures the full error body containing the ARM error detail is logged rather than a truncated version
+            log.error(errorMessage.append("Unable to save background search").toString() + " {}", e.contentUTF8());
+            throw handleFailureAndCreateException(errorMessage.toString(), armRpoExecutionDetailEntity, userAccount);
+        }
+
+        if (isNull(saveBackgroundSearchResponse)
+            || isNull(saveBackgroundSearchResponse.getStatus())
+            || !saveBackgroundSearchResponse.getStatus().equals(ARM_RPO_SUCCESS_RESPONSE_STATUS)) {
```

Comment:
The other code we have in this class (that calls the other ARM endpoints) is happy with any 2xx "success" response (any non-2xx response will throw the FeignException).

So do we need to explicitly check that the response code in the response body is 200 here? Might not be a bad idea, but perhaps we should either do it consistently across all the ARM calls, or not at all?